glossary.tex

```diff
@@ -268,3 +268,29 @@
                worden sind, sowie die die Menge der unbearbeiteten
                Änderungswünschen (Featurerequests).}
 }
+\newglossaryentry{Peer-to-Peer}
+{
+  name=Peer-to-Peer,
+  description={Begriff aus der Computer Netzwerkstruktur. Es beschreibt die
+               Verbindungsart zwischen zwei Computern in einem Netzwerk oder
+               Internet. Dabei kommunizieren zwei Teilnehmer über eine direkte
+               Verbindung miteinander, ohne das einer der beiden eine
```

Comment:
würde den letzten nebensatz weglassen und sowas wie:
"In einem Peer to Peer Netzwerk sind alle Teilnehmer gleichgestellt, die alternative ist das Client-Server Modell"

schreiben